cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/common/DefaultMacroEvaluator.java

```
@@ -90,6 +90,8 @@ public String evaluate(String macroFunction, String... arguments) throws Invalid
         return Bytes.toString(secureStore.getSecureData(namespace, arguments[0]).get());
       } catch (IOException e) {
         throw new InvalidMacroException("Failed to resolve macro '" + macroFunction + "(" + arguments[0] + ")'", e);
+      } catch (Exception e) {
+        e.printStackTrace();
```

propagate instead of printing stack trace?